app/src/main/java/com/limelight/AppView.java

```diff
@@ -386,6 +398,20 @@ public void run() {
             case CANCEL_ID:
                 return true;
 
+            case VIEW_DETAILS_ID:
+                AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setMessage(getResources().getString(R.string.applist_details_id) + " " + app.app.getAppId())
```

Comment:
Can you use `Dialog.displayDialog()` for app and PC details? That way we'll get an OK button and a title (which you can just make "App Details" and "PC Details" in strings.xml).